### ISSUE TITLE
[Bug] To correctly display remaining time

### ISSUE THREAD

**Issue by FlorianLatapie** [myfix16/BatteryTracker#9]
**Describe the bug**
Remaining time is negative when charging

**To Reproduce**
Steps to reproduce the behaviour:
1. Have less than 20%
2. Charge 
3. See error on notification. 

**Expected behaviour**
Compute remaining time for charging to 100%

**Screenshots**
![battery tracker issue](https://user-images.githubusercontent.com/70631622/212187675-a29ebc8c-5fa0-4440-a89f-51fe50e33e73.png)

**Desktop (please complete the following in formation):**
 - OS Version: Windows 11 22H2

**Comment by myfix16** [comment]
Yes, I also encountered this bug. I think it's related to the WindowsAPICodePack.dll I reference in my code.

I am porting the application to Windows App SDK and will remove this reference. I hope that will resolve the bug.

**Comment by FlorianLatapie** [comment]
OK I understand, thanks a lot !

**Comment by myfix16** [comment]
Hi, I have released BatteryTracker v1.3, which added new features and fixed lots of bugs. However, due to Windows App SDK's issue, I temporarily disabled displaying remaining time. I will reopen this issue and use it to track future updates.
